src/containers/GlossPage/components/ExampleField.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Input, Select, FieldSplitter } from '@ndla/forms';
+import { IGlossExample } from '@ndla/types-backend/concept-api';
+import { useTranslation } from 'react-i18next';
+import styled from '@emotion/styled';
+import TranscriptionsField from './TranscriptionsField';
+import { LANGUAGES } from '../glossData';
+import FormikField from '../../../components/FormikField';
+
+interface Props {
+  example: IGlossExample;
+  index: number;
+  name: string;
+}
+
+const StyledFormikField = styled(FormikField)`
+  margin: 0px;
+`;
+
+const ExampleField = ({ example, name }: Props) => {
```

Comment:
Generelt sett ikke så stor fan av at vi operer på objekter, og ikke faktiske felter her.

Reply:
Tenker du at vi burde definere alle elementer separat som formik fields? isåfall blir det veldig mange feilmeldinger å vise for eksempler ..

Reply:
Et eksempel er en sammenhengende enhet her, så tenker vi kan gå videre med dette og heller justere i etterkant når vi har fått dette ut i test.